media-sound/cadence/cadence-9999.ebuild

```diff
@@ -0,0 +1,86 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit git-r3 python-single-r1 xdg desktop
+
+DESCRIPTION="Collection of tools useful for audio production"
+HOMEPAGE="http://kxstudio.linuxaudio.org/Applications:Cadence"
+EGIT_REPO_URI="https://github.com/falkTX/Cadence.git"
+KEYWORDS=""
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="a2jmidid ladish -pulseaudio opengl"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+CDEPEND="
+	${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		dev-python/dbus-python[${PYTHON_MULTI_USEDEP}]
+		dev-python/PyQt5[dbus,gui,opengl?,svg,widgets,${PYTHON_MULTI_USEDEP}]
+	')
+	media-sound/jack2[dbus]
+	media-sound/jack_capture
+	a2jmidid? ( media-sound/a2jmidid[dbus] )
+	ladish? ( >=media-sound/ladish-9999 )
+	pulseaudio? ( media-sound/pulseaudio[jack] )
+"
+RDEPEND="${CDEPEND}"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	sed -i -e "s/python3/${EPYTHON}/" \
+		data/cadence \
+		data/cadence-aloop-daemon \
+		data/cadence-jacksettings \
+		data/cadence-logs \
+		data/cadence-render \
+		data/cadence-session-start \
+		data/catarina \
+		data/catia \
+		data/claudia \
+		data/claudia-launcher || die "sed failed"
+
+	default
+}
+
+src_compile() {
+	myemakeargs=(PREFIX="${EPREFIX}/usr"
+		SKIP_STRIPPING=true
+	)
+
+	emake "${myemakeargs[@]}"
+}
+
+src_install() {
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${ED}" install
+
+	python_fix_shebang ${ED}
+
+	# Clean up stuff that shouldn't be installed
+	rm -rf "${ED}"/etc/X11/xinit/xinitrc.d/61cadence-session-inject
+	rm -rf "${ED}"/etc/xdg/autostart/cadence-session-start.desktop
+	rm -rf "${ED}"/usr/share/applications/*.desktop
+
+	if use !pulseaudio; then
+		rm -rf "${ED}"/usr/bin/cadence-pulse2{jack,loopback}
+		rm -rf "${ED}"/usr/share/cadence/pulse2{jack, loopback}
+	fi
+	if use !ladish; then
+		rm -rf "${ED}"/usr/bin/claudia{,-launcher}
+		rm -rf "${ED}"/usr/share/cadence/icons/claudia-hicolor/
+	fi
+
+	# Replace desktop entries with QA issues with these
+	make_desktop_entry cadence Cadence cadence "AudioVideo;AudioVideoEditing;Qt"
+	make_desktop_entry catia Catia catia "AudioVideo;AudioVideoEditing;Qt"
+	if use ladish; then
+		make_desktop_entry claudia Claudia claudia "AudioVideo;AudioVideoEditing;Qt"
+		make_desktop_entry claudia-launcher "Claudia Launcher" claudia-launcher "AudioVideo;AudioVideoEditing;Qt"
+	fi
+	make_desktop_entry catarina Catarina catarina "AudioVideo;AudioVideoEditing;Qt"
```

Comment:
I'd prefer to leave catarina out or alternatively put it behind a USE flag. Most users won't need it and it's just noise in the application list this if it's present.

Reply:
Hmm, I think, we should keep it. Yes, of course, this is a test tool. However, the upstream Makefile also builds/installs it by default, therefore there might be users that want to use it and it should be the choice of the user whether to use or ignore it.
I'm okay with putting it behind a useflag.

Reply:
OK, USE flag is fine, guess we can just use one called `catarina`.